Description: Upstream changes introduced in version 0.6~mactel1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 macfanctld (0.6~mactel1) natty; urgency=low
 .
   * Use PnP rather than hardcoding resources and devices:
     locate applesmc via /sys/class/hwmon and check that device/name
     is "applesmc" as suggested by Julien BLACHE.
 .
 The person named in the Author field signed this changelog entry.
Author: Mikael Strom <mikael@sesamiq.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- macfanctld-0.6~mactel1.orig/macfanctld.1
+++ macfanctld-0.6~mactel1/macfanctld.1
@@ -1,4 +1,4 @@
-.TH macfanctld 1 "November 17, 2010" "Version 0.5" "USER COMMANDS"
+.TH macfanctld 1 "October 20, 2011" "Version 0.6" "USER COMMANDS"
 .SH NAME
 macfanctld \- Fan control for MacBook
 .SH SYNOPSIS
--- /dev/null
+++ macfanctld-0.6~mactel1/README
@@ -0,0 +1 @@
+See the manual page macfanctld (1) for more information.
